packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info-overlay/collections-info-overlay.component.ts

```diff
@@ -13,9 +13,12 @@ import { EventDisplayService } from '../../../../services/event-display.service'
 })
 export class CollectionsInfoOverlayComponent implements OnInit {
   @Input() showObjectsInfo: boolean;
+  hideInvisible: boolean;
```

Comment:
Just having this `hideInvisible` for keeping the checkbox value is enough. No need to make any other changes to the component code, you could just use the `isCut` of an object property to decide whether to display a row or not.